src/shogun/base/SGObject.h

```diff
@@ -448,6 +448,10 @@ class CSGObject
 	void put(const std::string& name, T* value)
 	{
 		put(Tag<T*>(name), value);
+
+		std::string callback_fun = name + "_callback";
```

Comment:
@gf712 do you think we could somehow define in AnyParameters that a given parameter has dependent parameters? :D

Reply:
hmmm I am not sure. We could add a vector of reference wrappers to other parameters? But why do you need it?